### PR TITLE
[FLINK-6932] [doc] Update inaccessible Dataflow Model paper link

### PR DESCRIPTION
**Pull request by zhangminglei**
Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [ ] General
  - The pull request references the related JIRA issue ("[FLINK-XXX] Jira title text")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [ ] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

File: docs/dev/event_time.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ to use timestamp assignment and watermark generation in the Flink DataStream API
 *Note: Flink implements many techniques from the Dataflow Model. For a good introduction to event time and watermarks, have a look at the articles below.*
 
   - [Streaming 101](https://www.oreilly.com/ideas/the-world-beyond-batch-streaming-101) by Tyler Akidau
-  - The [Dataflow Model paper](https://static.googleusercontent.com/media/research.google.com/en/pubs/archive/43864.pdf)
```
Comment (greghogan) [inline docs/dev/event_time.md]: Looks like this should have been `https://research.google.com/pubs/archive/43864.pdf` before the redirect (linked from `https://research.google.com/pubs/pub43864.html`)

Comment (zhangminglei) [inline docs/dev/event_time.md]: Yes. Exactly. I will do a change to this.

**Comment by zhangminglei** [comment]
cc @zentol Please take a look. Thank you so much. :cake:

**Comment by tzulitai** [comment]
LGTM, thanks for the fix @zhangminglei. Merging ...

**Pull request closed without merging**